### PR TITLE
Covid 19 Tracker app

### PR DESCRIPTION
**Pull request by Hadi111jafari**
In this app, I did the following:

- Selected COVID-19 API
- build files structure for the React app
- Configured Routes
- Fetched API and rendered the list
- Create tests for the application
- Add styles for components
- deployed in GitHub pages
- recorded a video

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,104 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team for guidance
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (elmar8287) [inline README.md]: - [ ] Please, consider fixing the link to the MIT.md file, actually, it is broken.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,104 @@
...
+
+Feel free to check the [issues page](https://github.com/Hadi111jafari/covid19-tracker/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team for guidance
```
Comment (elmar8287) [inline README.md]: - [ ] Please, consider that in the "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/App.js [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+  <Router>
+    <header>
+      <Navbar />
+    </header>
+    <Routes>
+      <Route exact path="/" element={<CountriesCountainer />} />
+      <Route path="/:country" element={<Region />} />
+    </Routes>
+  </Router>
+);
```
Comment (elmar8287) [inline src/App.js]: - [ ] Please, consider that when the app opens the first time it is not opening the right page. As you can see on the gif, I have to click back to see the list of countries. On the first opened page the "loading" status is available. Let us fix this issue to cover the project requirements 👍

File: src/__tests__/__snapshots__/countryContainer.test.js.snap [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (elmar8287) [inline src/__tests__/__snapshots__/countryContainer.test.js.snap:1]: - [ ] Please, consider fixing the not passed test below:
Screen Shot 2022-05-27 at 10 31 10

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

I want to highlight:

- Nice design and functionality
- Nice video presentation
- Nice PR description
- Nice commits history 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/__snapshots__/countryContainer.test.js.snap (hunk 4 above)
Comment (Hadi111jafari) [reply src/__tests__/__snapshots__/countryContainer.test.js.snap:1]: Thank you for your nice feedback now test are passing.
screen shot

File: README.md (hunk 1 above)
Comment (Hadi111jafari) [reply README.md]: Thank you for your feedback, I fixed the link

File: README.md (hunk 2 above)
Comment (Hadi111jafari) [reply README.md]: I am happy you reminded me what I forgot to make my project complete
the issue is fixed.
thank you.

File: src/__tests__/__snapshots__/countryContainer.test.js.snap (hunk 4 above)
Comment (Hadi111jafari) [reply src/__tests__/__snapshots__/countryContainer.test.js.snap:1]: Screenshot from 2022-05-27 16-29-18
Thank you for your feedback, I implemented requested changes.
Here is the test screenshot that is passing

File: src/App.js (hunk 3 above)
Comment (Hadi111jafari) [reply src/App.js]: HI @elmar8287 
Thank you, for your feedback, I have implemented requested changes.

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**